src/ocean/core/BitArray.d

```diff
@@ -189,13 +191,8 @@ struct BitArray
     }
     body
     {
-        size_t mDim=len/32;
-        ptr[0..mDim] = rhs.ptr[0..mDim];
```

Comment:
Based on this line I'm likely to think we're missing something that we have removed with `if (rest)` branch. The reason for that is that this line `ptr[0..mDim] = rhs.ptr[0..mDim]` will not copy stuff bit by bit, but it will actually perform `memcpy` or similar thing inside runtime.